github3/repos/repo.py

```diff
@@ -396,21 +396,31 @@ def branch(self, name):
         json = None
         if name:
             url = self._build_url('branches', name, base_url=self._api)
-            json = self._json(self._get(url), 200)
+            headers = {'Accept': 'application/vnd.github.loki-preview+json'}
```

Comment:
So for exactly the reason that I refuse to duplicate these headers, please add a `CUSTOM_HEADERS` or `PREVIEW_HEADERS` class attribute to the `Branch` class and then do:

```py
json = self._json(self._get(url, headers=Branch.CUSTOM_HEADERS), 200)
```